.circleci/docker/common/install_conda.sh

```diff
@@ -102,6 +102,7 @@ if [ -n "$ANACONDA_PYTHON_VERSION" ]; then
   # TODO: Why is scipy pinned
   # Pin MyPy version because new errors are likely to appear with each release
   # Pin hypothesis to avoid flakiness: https://github.com/pytorch/pytorch/issues/31136
+  # Pin coverage so we can use COVERAGE_RCFILE
```

Comment:
Does this mean that we expect `COVERAGE_RCFILE` to be removed in future versions of `coverage`? Or is the purpose of this just to modify the script in some way so that the Docker image gets rebuilt?

Reply:
Ah, no, this was because our previously installed version of coverage did not have the COVERAGE_RCFILE option, so this explains why we pinned it in the first place